src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalsAndReturnsTests.cs

```diff
@@ -3850,13 +3852,24 @@ public void M(int[] array, ref int value)
                 // (6,9): error CS8373: The left-hand side of a ref assignment must be a ref local or parameter.
                 //         array[0] = ref value;
                 Diagnostic(ErrorCode.ERR_RefLocalOrParamExpected, "array[0]").WithLocation(6, 9));
+
+            var tree = compilation.SyntaxTrees.Single();
```

Comment:
Consider creating a helper method, since the verification pattern is repeated in multiple tests.

Reply:
Actually most of it is not repeated. They're casted to different types, accessing different members. I'd prefer to keep it this way for readability.